src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -704,6 +704,12 @@
     text: az acr manifest list-metadata -r myregistry -n hello-world
   - name: List the metadata of the manifests in the repository 'hello-world'.
     text: az acr manifest list-metadata myregistry.azurecr.io/hello-world
+  - name: List the metadata of all manifests (tagged/untagged) in the repository 'hello-world'.
+    text: az acr manifest list-metadata -r myregistry -n hello-world --query "[]"
```

Comment:
is the query necessary? how does that affect the results?

Reply:
The comments for the work item say the previous command included untagged artifacts too. I can remove the 3rd case but would recommend both all / only untagged queries.

Reply:
The list-metadata command only displays tagged artifacts should we add verbiage to that command?